src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -57,7 +82,7 @@ public PythonEngineVersion Engine
                 if (engine != value)
                 {
                     engine = value;
-                    RaisePropertyChanged(nameof(Engine));
+                    RaisePropertyChanged(nameof(EngineName));
```

Comment:
is a public setter useful?

Reply:
I can revert this change...The parent property is obsolete anyway

Reply:
It is bound